llvm/test/Verifier/intrinsic-cmp.ll

```diff
@@ -0,0 +1,26 @@
+; RUN: not opt -S -passes=verify 2>&1 < %s | FileCheck %s
+
+
+define void @matching_vector_lens(<4 x i32> %arg1, <4 x i32> %arg2) {
+  ; CHECK-LABEL: cmp_vector_lens_match
```

Comment:
I don't think this is how  the errors are formatted. You should be getting the error message and *then* the intrinsic name, I think.

You should try running opt on this test locally and then adjust the check lines to match the output.